src/analysis/souffle/tests/arcs_fact_tests/ok_claim_propagates.dl

```diff
@@ -0,0 +1,32 @@
+// claim propagates through a chain of particles
+// #Ingress: P1
+// #OK
+//particle P1
+//  foo: writes Foo {}
+//  claim foo is trusted
+//particle P2
+//  bar: reads Foo {}
+//  foo: writes Foo {}
+//particle P3
+//  bar: reads Foo {}
+//  check bar is trusted
+//recipe R
+//  P1
+//    foo: writes h1
+//  P2
+//    bar: reads h1
+//    foo: writes h2
+//  P3
+//    bar: reads h2
+#include "taint.dl"
+#include "analyze_tag_checks_test_helper.dl"
+
```

Comment:
nit: just add a comment that this comes from the manifest (or something to that effect).

Reply:
Done.